Formula/heksa.rb

```diff
@@ -0,0 +1,29 @@
+class Heksa < Formula
+  desc "CLI hex dumper with colors"
+  homepage "https://github.com/raspi/heksa"
+  url "https://github.com/raspi/heksa.git",
+      :tag      => "v1.12.1",
+      :revision => "54a68a2910a87b8b9cfcda28b8693e18dd9e2463"
```

Comment:
This is intentional because as part of the build process git revision gets queried.